admin-dev/themes/new-theme/js/vue/pages/product/ProductImageUpload.vue

```diff
@@ -79,7 +79,9 @@
   };
 
   async function uploadImages(fileList) {
-    return fetch(router.generate('admin_products_v2_images'), {
+    return fetch(router.generate('admin_products_v2_images', {
+      productId: 1//@todo: where do i store product id?
```

Comment:
Where should i store productId for js?
it could be data attr, but not sure to what element it could be attached.

Reply:
Meh, if this is a single component, I would use a data attr to pass it as a prop